### PR TITLE
Add TEAM_SERVICES env to otomi-console chart

### PR DESCRIPTION
**Pull request by svatwork**
Closes https://github.com/redkubes/unassigned-issues/issues/229.

I assume this is sufficient, the output is:

```
            - name: TEAM_SERVICES
              value: "[{\"auth\":true,\"enabled\":true,\"host\":\"harbor\",\"isShared\":true,\"name\":\"harbor\",\"namespace\":\"harbor\",\"svc\":\"harbor-portal\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"host\":\"httpbin\",\"isShared\":true,\"name\":\"httpbin\",\"namespace\":\"shared\",\"svc\":\"httpbin\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"host\":\"vault\",\"isShared\":true,\"logo\":{\"name\":\"vault\"},\"name\":\"vault\",\"namespace\":\"vault\",\"path\":\"ui/vault/secrets/secret/list/teams/#NS#/\",\"port\":8200,\"svc\":\"vault-0\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"host\":\"kubeapps\",\"isShared\":true,\"logo\":{\"name\":\"kubernetes\"},\"name\":\"kubeapps\",\"namespace\":\"kubeapps\",\"svc\":\"kubeapps\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"host\":\"grafana\",\"name\":\"loki\",\"path\":\"explore?orgId=1\\u0026left=%5B%22now-1h%22,%22now%22,%22Loki%22,%7B%22expr%22:%22%7Bnamespace%3D%5C%22#NS#%5C%22%7D%22%7D,%7B%22mode%22:%22Logs%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D\",\"svc\":\"po-grafana\",\"type\":\"public\"},{\"auth\":true,\"authz\":{\"workload\":{\"app.kubernetes.io/instance\":\"prometheus-__TEAM\",\"app.kubernetes.io/name\":\"__TEAM-po-grafana\"}},\"enabled\":true,\"hasPrefix\":true,\"name\":\"grafana\",\"svc\":\"po-grafana\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"hasPrefix\":true,\"name\":\"prometheus\",\"port\":9090,\"svc\":\"po-prometheus\",\"type\":\"public\"},{\"auth\":true,\"enabled\":true,\"hasPrefix\":true,\"logo\":{\"name\":\"prometheus\"},\"name\":\"alertmanager\",\"port\":9093,\"svc\":\"po-alertmanager\",\"type\":\"public\"}]"
```

Apparently there is no limit to a Linux (env) variable, it depends on the space allocated (see e.g. https://stackoverflow.com/a/1078125/8357826). 

But maybe with the network request it might be a problem, let's just build it and see what happens?

### CODE REVIEW

**Comment by svatwork** [comment]
It is hard to reach a limit, see e.g.: 

```
wc -c otomi-api/test/env/teams/services.otomi.yaml                                                                 
2662 otomi-api/test/env/teams/services.otomi.yaml
```

Only 2662 byes (2.6KB)

**Comment by Morriz** [comment]
First of all I see you mount core services, not team services. Secondly, I prefer filtering of what is needed only, as filemounts are unneeded noise.

**Review by Morriz — CHANGES_REQUESTED** [review]
See comments for requested changes

**Comment by svatwork** [comment]
> First of all I see you mount core services, not team services. Secondly, I prefer filtering of what is needed only, as filemounts are unneeded noise.

Did you look at the code in this PR? If so, how can `$teamServices` be "core services"? To my understanding `$services` are core services?

If not, where else can I find team services if not `$teamServices`?

File: values/otomi-console/otomi-console.gotmpl [hunk 1]
```diff
@@ -11,14 +11,29 @@
...
+{{- $teamServices := list -}}
+  {{- if hasKey $v.teamConfig "teams" -}}
+    {{- range $teamId, $team := $v.teamConfig.teams }}
+      {{- if hasKey $team "services" -}}
+        {{- range $s := $team.services -}}
+          {{- $sd := dict -}}
+            {{- with $s.name -}}{{ $sd = set $sd "name" . }}{{- end }}
+            {{- with $s.type -}}{{ $sd = set $sd "type" . }}{{- end }}
+            {{- if hasKey $s "ksvc" -}}{{ $sd = set $sd "ksvc" true  }}{{- end }}
+            {{- with $s.port -}}{{ $sd = set $sd "port" . }}{{- end }}
```
Comment (Morriz) [inline values/otomi-console/otomi-console.gotmpl:31]: does this idd contain all the values needed for the console?

**Review by Morriz — APPROVED** [review]

**Pull request merged**